lib/game.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+class Game #:nodoc:
+  attr_reader :space_used
+  attr_reader :player
+  def initialize
+    @board = [%w[_ _ _], %w[_ _ _], %w[_ _ _]]
+    @player = 1
+    @space_used = 0
+    @moves_player1 = ''
+    @moves_player2 = ''
+  end
+
+  def check_for_win(player_flag)
+    winners = %w[123 456 789 147 258 369 159 357]
+    moves = player_flag == 1 ? @moves_player2 : @moves_player1
+    moves = moves[moves.length - 4, moves.length - 1] if moves.length > 3
+    winners.include?(moves)
+  end
+
+  def display_board
+    @board.each_with_index do |row, row_index|
+      row_string = ' '
+      row.each_with_index do |element, index|
+        row_string += ' | ' if index != 0
+        row_string += element
+      end
+      puts row_string
```

Comment:
Game logic should be kept in classes (no “puts” and “get.chomp” there!)
User interaction commands should be in one executable file main.rb